media-gfx/blender/blender-4.4.3-r1.ebuild

```
@@ -0,0 +1,900 @@
+		-DWITH_OPENIMAGEDENOISE="$(usex oidn)"
+		-DWITH_OPENSUBDIV="$(usex opensubdiv)"
+		-DWITH_OPENVDB="$(usex openvdb)"
+		-DWITH_OPENVDB_BLOSC="$(usex openvdb)"
+		-DWITH_POTRACE="$(usex potrace)"
+		-DWITH_PUGIXML="$(usex pugixml)"
+		# -DWITH_QUADRIFLOW=ON
+		-DWITH_RENDERDOC="$(usex renderdoc)"
+		-DWITH_TBB="$(usex tbb)"
+		-DWITH_UNITY_BUILD="no"
+		-DWITH_USD="no" # TODO: Package USD
+		-DWITH_VULKAN_BACKEND="$(usex vulkan)" # experimental
+		-DWITH_XR_OPENXR="no"
+
+		-DWITH_SYSTEM_BULLET="yes"
+		-DWITH_SYSTEM_EIGEN3="yes"
+		-DWITH_SYSTEM_FREETYPE="yes"
+		-DWITH_SYSTEM_GFLAGS="yes"
+		-DWITH_SYSTEM_GLOG="yes"
+		-DWITH_SYSTEM_LZO="yes"
+
+		# Compiler Options:
+		# -DWITH_BUILDINFO="yes"
+		-DWITH_OPENMP="$(usex openmp)"
+
+		# System Options:
+		-DWITH_INSTALL_PORTABLE="no"
+		-DWITH_MEM_JEMALLOC="$(usex jemalloc)"
+		-DWITH_MEM_VALGRIND="$(usex valgrind)"
+
+		# GHOST Options:
+		-DWITH_GHOST_WAYLAND="$(usex wayland)"
+		# -DWITH_GHOST_WAYLAND_APP_ID="blender-${BV}"
+		-DWITH_GHOST_WAYLAND_DYNLOAD="no"
+		-DWITH_GHOST_X11="$(usex X)"
+		# -DWITH_GHOST_XDND=ON
+		# -DWITH_X11_XF86VMODE=ON
+		# -DWITH_X11_XFIXES=ON
+		# -DWITH_X11_XINPUT=ON
+		# -DWITH_GHOST_WAYLAND_DYNLOAD # visible wayland?
+		# -DWITH_GHOST_WAYLAND_LIBDECOR # visible wayland?
+
+		# Image Formats:
+		# -DWITH_IMAGE_CINEON=ON
+		-DWITH_IMAGE_OPENEXR="$(usex openexr)"
+		-DWITH_IMAGE_OPENJPEG="$(usex jpeg2k)"
+		-DWITH_IMAGE_WEBP="$(usex webp)" # unlisted
+
+		# Audio:
+		# -DWITH_AUDASPACE=OFF
+		# -DWITH_SYSTEM_AUDASPACE=OFF
+		-DWITH_CODEC_FFMPEG="$(usex ffmpeg)"
+		-DWITH_CODEC_SNDFILE="$(usex sndfile)"
+		# -DWITH_COREAUDIO=OFF
+		-DWITH_JACK="$(usex jack)"
+		# -DWITH_JACK_DYNLOAD=
+		-DWITH_OPENAL="$(usex openal)"
+		-DWITH_PIPEWIRE="$(usex pipewire)"
+		# -DWITH_PIPEWIRE_DYNLOAD=
+		-DWITH_PULSEAUDIO="$(usex pulseaudio)"
+		# -DWITH_PULSEAUDIO_DYNLOAD=
+		-DWITH_SDL="$(usex sdl)"
+		# -DWITH_WASAPI=OFF
+
+		# Python:
+		# -DWITH_PYTHON=ON
+		-DWITH_PYTHON_INSTALL="no"
+		-DWITH_PYTHON_INSTALL_NUMPY="no"
+		-DWITH_PYTHON_INSTALL_ZSTANDARD="no"
+		# -DWITH_PYTHON_MODULE="no"
+		-DWITH_PYTHON_SAFETY="OFF"
+		-DWITH_PYTHON_SECURITY="yes"
+		-DPYTHON_INCLUDE_DIR="$(python_get_includedir)"
+		-DPYTHON_LIBRARY="$(python_get_library_path)"
+		-DPYTHON_VERSION="${EPYTHON/python/}"
+		-DWITH_DRACO="yes" # TODO: Package Draco # NOTE use bundled for now
+
+		# Modifiers:
+		-DWITH_MOD_FLUID="$(usex fluid)"
+		# -DWITH_MOD_REMESH=ON
+		-DWITH_MOD_OCEANSIM="$(usex fftw)"
+
+		# Rendering:
+		-DWITH_HYDRA="no" # TODO: Package Hydra
+
+		# Rendering (Cycles):
+		-DWITH_CYCLES_OSL="$(usex osl)"
+		-DWITH_CYCLES_EMBREE="$(usex embree)"
+		-DWITH_CYCLES_PATH_GUIDING="$(usex openpgl)"
+
+		-DWITH_CYCLES_DEVICE_OPTIX="$(usex optix)"
+		-DWITH_CYCLES_DEVICE_CUDA="$(usex cuda)"
+		-DWITH_CYCLES_CUDA_BINARIES="$(usex cuda "$(usex cycles-bin-kernels)")"
+
+		-DWITH_CYCLES_DEVICE_HIP="$(usex hip)"
+		-DWITH_CYCLES_HIP_BINARIES="$(usex hip "$(usex cycles-bin-kernels)")"
```

```suggestion
		-DWITH_CYCLES_HIP_BINARIES="$(usex hip "$(usex cycles-bin-kernels)")"
		-DWITH_CYCLES_DEVICE_HIPRT="$(usex hip "$(usex hiprt)")"
```

Cycles options are here.